### ISSUE TITLE
please upgrade deno core -> 0.278.0

### ISSUE THREAD

**Issue by i18n-now** [rscarson/rustyscript#84]
<img width="1728" alt="image" src="https://github.com/rscarson/rustyscript/assets/145643935/d710a0d2-bcf3-44bb-a7ca-043667072ad5">

```

 --> src/ext/fs.rs:3:5
  |
3 | use deno_io::fs::FsError;
  |     ^^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

error[E0308]: mismatched types
   --> src/main.rs:20:24
    |
20  |         extensions: vec![deno_fs::deno_fs::init_ops_and_esm::<ext::fs::Permissions>(
    |  ________________________^
21  | |         fs.clone(),
22  | |       )],
    | |_______^ expected `Extension`, found a different `Extension`
    |
    = note: `Extension` and `Extension` have similar names, but are actually distinct types
note: `Extension` is defined in crate `deno_core`
   --> /Users/z/.cargo/registry/src/index.crates.io-6f17d22bba15001f/deno_core-0.278.0/extensions.rs:582:1
    |
582 | pub struct Extension {
    | ^^^^^^^^^^^^^^^^^^^^
note: `Extension` is defined in crate `deno_core`
   --> /Users/z/.cargo/registry/src/index.crates.io-6f17d22bba15001f/deno_core-0.238.0/extensions.rs:491:1
    |
491 | pub struct Extension {
    | ^^^^^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `deno_core` are being used?
```

**Comment by rscarson** [comment]
I have updated the core for the next version
Unfortunately that version changes a lot of deno_core's API, breaking a number of features
I will need some time to come up with a workaround

**Comment by rscarson** [comment]
Fixed in version `0.4.0`
